src/index.html

```diff
@@ -22,6 +22,388 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a
+          href="index.html"
+          class="logo__link"
+        >
+          <img
+            src="./images/logo.png"
+            alt="MOYO logo"
+            class="logo__image"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="##"
```

Comment:
The href attribute value '##' is not a valid URL. Consider using '#' for a placeholder or providing a valid URL.